index.js

```diff
@@ -1 +1 @@
-module.exports = require('./dist/less-syntax');
+module.exports = require('./lib/less-syntax');
```

Comment:
These files will be generated Gulp. See https://github.com/webschik/postcss-less/blob/master/gulpfile.babel.js#L25